### PR TITLE
add paramater basic configuration

### PR DESCRIPTION
**Pull request by alavieille**
[OO-FEATURE] add default max age to 600s in block form strategies
[OO-BCBREAK] Adding new parameter open_orchestra_backoffice.block_default_configuration for block form strategies

https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/278
https://github.com/open-orchestra/open-orchestra-elastica-bundle/pull/52
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1843

### CODE REVIEW

File: MediaAdminBundle/GenerateForm/Strategies/GalleryStrategy.php [hunk 1]
```diff
@@ -19,11 +19,13 @@ class GalleryStrategy extends AbstractBlockStrategy
     protected $thumbnailConfig = array();
 
     /**
+     * @param array               $basicConfigurationBlock
      * @param TranslatorInterface $translator
      * @param array               $thumbnailConfig
      */
-    public function __construct(TranslatorInterface $translator, array $thumbnailConfig)
+    public function __construct(array $basicConfigurationBlock, TranslatorInterface $translator, array $thumbnailConfig)
```
Comment (ngilain) [inline MediaAdminBundle/GenerateForm/Strategies/GalleryStrategy.php]: @alavieille : basicBlockConfiguration

**Comment by alavieille** [comment]
@ngilain pr updated

File: MediaAdminBundle/GenerateForm/Strategies/GalleryStrategy.php [hunk 2]
```diff
@@ -19,11 +19,13 @@ class GalleryStrategy extends AbstractBlockStrategy
     protected $thumbnailConfig = array();
 
     /**
+     * @param array               $basicBlockConfiguration
      * @param TranslatorInterface $translator
      * @param array               $thumbnailConfig
      */
-    public function __construct(TranslatorInterface $translator, array $thumbnailConfig)
+    public function __construct(array $basicBlockConfiguration, TranslatorInterface $translator, array $thumbnailConfig)
```
Comment (ngilain) [inline MediaAdminBundle/GenerateForm/Strategies/GalleryStrategy.php]: @alavieille : $thumbnailConfiguration or basicBlockConfig to have the same terminology

**Comment by ngilain** [comment]
:+1: with or without the last renaming

**Pull request merged**